### PR TITLE
Provided configuration for controlling offset monitor logging

### PR DESCRIPTION
**Pull request by bbaugher**

### CODE REVIEW

**Comment by noslowerdna** [comment]
+1

**Comment by bbaugher** [comment]
In order to get everything to work I had to include an option to download and supply the log4j implementation jar. In later version of the offset monitor (not released) log4j jar is included with the offset monitor jar

**Comment by bbaugher** [comment]
Not sure why travis says the builds failed as you can see all the tests pass

**Comment by gwhitsitt** [comment]
+1

File: attributes/default.rb [hunk 1]
```diff
@@ -79,7 +79,19 @@
 default["kafka"]["offset_monitor"]["refresh"] = "15.minutes"
 default["kafka"]["offset_monitor"]["retain"] = "7.days"
 
+default["kafka"]["offset_monitor"]["include_log4j_jar"] = true
```
Comment (swarnim87) [inline attributes/default.rb]: Should some of these be defined in the README file?

**Comment by swarnim87** [comment]
+1 after the minor comment.

File: attributes/default.rb (hunk 1 above)
Comment (bbaugher) [inline attributes/default.rb]: Missed this one added it to the README, https://github.com/cerner/cerner_kafka/pull/52/commits/5b5c303a750baa89884be467aed3358cc60fde4e

**Comment by bbaugher** [comment]
Ended up making a couple other changes after realizing what might happen to our existing installations. 

Previously the classpath for the offset monitor referenced the jar itself and we would land the jar with the version in the name. The change here is to include the entire offset monitor install directory on the classpath to have it include our log4j config and possibly log4j jar. However this would mean if you updated to a new version of the offset monitor there would be 2 offset monitor jars in the install directory / classpath since the file name would contain the version. The change then is to keep the offset monitor jar file name the same and leave it up to Chef to update it appropriately.

The next problem is older installations (prior to this change) would have the same problem 2 offset monitor jars on the classpath. The simplest fix for this I could come up with was to change the default value for the offset monitor installation directory.

Let me know if you have any problems with this plan. Theres other ways to get around these issues but this seemed pretty easy.

**Comment by noslowerdna** [comment]
+1

**Comment by dhondgepooja** [comment]
+1

**Comment by bbaugher** [comment]
Squashed. Tests passed, merging

**Pull request merged**